### ISSUE TITLE
Story: organize provider MARC profile info/data

### ISSUE THREAD

**Issue by corylown** [pod4lib/aggregator#669]

**Comment by JohnMarkOckerbloom** [comment]
Initially, I'm thinking this may simply involve linking to MARC profile documentation from the Provider Info tag, and possibly providing storage for documentation that's not already online somewhere.  (There is documentation online for most institutions, in many cases in Google docs, but in some cases on the provider's own website.)

I may shortly request an additional form field under the "Provider Details" tab on the Manage Organizations page to support this, probably as a separate ticket for that specific functionality.

There may be further development or organization of this documentation in the future (James Robinson has suggested we might be able to make it more structured), but links to documentation files seem to be  a reasonable first cut at this functionality, and may be what we can manage in this work cycle.  We might develop it more if we have time, though.  Comments from others welcome.

**Comment by JohnMarkOckerbloom** [comment]
At this point, I'm waiting on #565 to be implemented to provide the basic functionality we'll have in this workcycle.  Once the form is there, I can populate it with links to the pages I know about, either the one at Harvard or the ones on Google docs that James has told me about,
